eras/conway/test-suite/src/Test/Cardano/Ledger/Conway/Serialisation/Roundtrip.hs

```diff
@@ -1,26 +1,34 @@
 {-# LANGUAGE AllowAmbiguousTypes #-}
+{-# LANGUAGE DataKinds #-}
 {-# LANGUAGE FlexibleContexts #-}
 {-# LANGUAGE RankNTypes #-}
 {-# LANGUAGE ScopedTypeVariables #-}
 {-# LANGUAGE TypeApplications #-}
 
 module Test.Cardano.Ledger.Conway.Serialisation.Roundtrip (allprops) where
 
+import Cardano.Ledger.Alonzo.TxWits (Redeemers)
+import Cardano.Ledger.Binary.Version (natVersion)
 import Cardano.Ledger.Conway.Genesis (ConwayGenesis (..))
 import Cardano.Ledger.Core (Era (..))
 import Data.Data (Proxy (..), typeRep)
-import Test.Cardano.Ledger.Binary.RoundTrip (roundTripCborExpectation)
+import Test.Cardano.Ledger.Binary.RoundTrip (
+  roundTripAnnExpectation',
+  roundTripCborExpectation,
+ )
 import Test.Cardano.Ledger.Conway.Serialisation.Generators ()
 import Test.Tasty (TestTree, testGroup)
 import Test.Tasty.QuickCheck (testProperty)
 
 allprops ::
   forall e.
-  ( Era e
-  ) =>
+  (Era e) =>
   TestTree
 allprops =
   testGroup
-    (show $ typeRep (Proxy @e))
-    [ testProperty "ConwayGenesis" $ roundTripCborExpectation @(ConwayGenesis (EraCrypto e))
+    ("Conway CBOR roundtrip " <> show (typeRep (Proxy @e)))
+    [ testProperty "ConwayGenesis" $
+        roundTripCborExpectation @(ConwayGenesis (EraCrypto e))
+    , testProperty "Redeemers" $
```

Comment:
The test we also need here is the embed test. Where we generate and serialize Babbage Redeemers , but deserialize as Conway. There are helper functions for testing this as well.  Checkout `embedTripExpectation`

Reply:
:exploding_head: Awesome! Thanks!